datafusion/proto/Cargo.toml

```diff
@@ -50,6 +50,7 @@ chrono = { workspace = true }
 datafusion = { workspace = true, default-features = true }
 datafusion-common = { workspace = true, default-features = true }
 datafusion-expr = { workspace = true }
+datafusion-physical-expr-common = { workspace = true }
```

Comment:
since proto already relies on the core datafusion crate this is not a new (direct) dependency.

Though maybe it highlights that it would be good to `pub use` the `AggregateExprBuilder` in `datafusion` so using it doesn't need a new explicit dependency 🤔